async_request.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |s|
   s.test_files = Dir['spec/**/*']
 
   s.add_dependency 'rails', '>= 4.2'
-  s.add_dependency 'sidekiq', '~> 4.0'
+  s.add_dependency 'sidekiq', '>= 4.2', '< 6'
```

Comment:
This is actually upgrading the sidekiq requirement. It should be `'>= 4.0', '< 6'`